addons/mass_mailing/models/mailing.py

```diff
@@ -223,6 +223,7 @@ def _get_default_mail_server_id(self):
     clicks_ratio = fields.Integer(compute="_compute_clicks_ratio", string="Number of Clicks")
     next_departure = fields.Datetime(compute="_compute_next_departure", string='Scheduled date')
     # UX
+    next_departure_is_past = fields.Boolean(compute="_compute_next_departure_is_past")
```

Comment:
Would compute both in the same method to simplify code
```suggestion
    next_departure = fields.Datetime(compute="_compute_next_departure", string='Scheduled date')
    next_departure_is_past = fields.Boolean(compute="_compute_next_departure", string="Scheduled is in the past)
    # UX
```